2017-12-16 17:44:01 [scrapy.core.downloader.handlers.http11] WARNING: Got data loss in http://gank.io/xiandu/wow/page/245. If you want to process broken responses set the setting DOWNLOAD_FAIL_ON_DATALOSS = False -- This message won't be shown in further requests
2017-12-16 17:50:44 [scrapy.core.downloader.handlers.http11] WARNING: Got data loss in http://gank.io/xiandu/wow/page/34. If you want to process broken responses set the setting DOWNLOAD_FAIL_ON_DATALOSS = False -- This message won't be shown in further requests
2017-12-16 18:06:00 [scrapy.core.downloader.handlers.http11] WARNING: Got data loss in http://www.qdaily.com/articles/48251.html?source=feed&utm_source=gank.io%2Fxiandu&utm_medium=website. If you want to process broken responses set the setting DOWNLOAD_FAIL_ON_DATALOSS = False -- This message won't be shown in further requests
2017-12-16 22:23:51 [scrapy.core.scraper] ERROR: Spider error processing <GET http://gank.io/xiandu> (referer: None)
Traceback (most recent call last):
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\Users\sgzhang\Desktop\python\myproject\python_spider\day09\myspider\myspider\spiders\xiandu.py", line 23, in parse
    meta={"item": item}
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\http\request\__init__.py", line 25, in __init__
    self._set_url(url)
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\http\request\__init__.py", line 58, in _set_url
    raise ValueError('Missing scheme in request url: %s' % self._url)
ValueError: Missing scheme in request url: /xiandu/view/engadget
2017-12-16 22:24:41 [scrapy.core.scraper] ERROR: Spider error processing <GET http://gank.io/xiandu> (referer: None)
Traceback (most recent call last):
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\utils\defer.py", line 102, in iter_errback
    yield next(it)
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "C:\Users\sgzhang\Desktop\python\myproject\python_spider\day09\myspider\myspider\spiders\xiandu.py", line 23, in parse
    meta={"item": item}
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\http\request\__init__.py", line 25, in __init__
    self._set_url(url)
  File "C:\Users\sgzhang\Anaconda3\lib\site-packages\scrapy\http\request\__init__.py", line 58, in _set_url
    raise ValueError('Missing scheme in request url: %s' % self._url)
ValueError: Missing scheme in request url: /xiandu/view/engadget
